stratum/glue/logging.cc

```diff
@@ -57,6 +58,11 @@ void InitStratumLogging() {
   if (FLAGS_logtostderr) {
     LogToStderr();
   }
+
+  LOG(INFO) << "Stratum version " << kBuildScmRevision << " ("
+            << kBuildScmStatus << ")"
+            << " built at " << kBuildTimestamp << " on host " << kBuildHost
```

Comment:
Consider a more human-readable timestamp; for example:
```suggestion
            << " built at " << absl::FromTimeT(kBuildTimestamp) << " on host " << kBuildHost
```

Reply:
And perhaps make this part of the log message conditional if `kBuildTimestamp > 0`

Reply:
Good idea, I had this in mind when writing the code but must have eluded me at some point.
For the conditional part a ternary operator expression could do it.

Reply:
Here is how a non-stamped build looks like:

`I20210120 23:16:39.726083 12592 logging.cc:63] Stratum version 0 (redacted) built at 1970-01-01T00:00:00+00:00 on host redacted by user redacted.`

Just ommiting the ts does not change much. Do we just omit the whole message? I'd actually like to keep it, as it's very obviously not an official release.

Reply:
`Stratum version - development (unstamped)`

Reply:
"0" is better than "1970-01-01T00:00:00+00:00"

Reply:
or turn on stamping by default :)